lib/grizzly/zwave/commands/switch_multilevel_report.ex

```diff
@@ -87,10 +87,10 @@ defmodule Grizzly.ZWave.Commands.SwitchMultilevelReport do
   end
 
   defp value_from_byte(0x00), do: {:ok, :off}
-  defp value_from_byte(byte) when byte in 0..99, do: {:ok, byte}
+  defp value_from_byte(byte) when byte in 0..100, do: {:ok, byte}
   defp value_from_byte(0xFE), do: {:ok, :unknown}
   # deprecated
-  defp value_from_byte(0xFF), do: {:ok, 99}
+  defp value_from_byte(0xFF), do: {:ok, 100}
```

Comment:
I am on the fence about this one and this is definitely nitpicky. I think we can in this particular decode to still decode to`99` as I think it is more in line with the language of the standard. A device that is sending `0xFF` is either old or out of line with spec so it really does not matter but the spec says it should be considered 100%, so if we are going to decode a byte value to mean 100% I think it is best what decode to 100% of what the spec says which is `99`.

Reply:
Agreed. Fixed.